From 3d7e084d8bcf4279c73ac672ec0f51ad74dbe92a Mon Sep 17 00:00:00 2001
From: Jeremy Newton <Jeremy.Newton@amd.com>
Date: Tue, 25 Jan 2022 21:44:06 -0500
Subject: [PATCH 1/2] Fix cmake file location

This was found while attempting to package for Fedora. Fedora and most
rpm distros expect the cmake file to be installed into
${CMAKE_INSTALL_LIBDIR}/cmake, instead of hardcoded to lib/cmake.

Signed-off-by: Jeremy Newton <Jeremy.Newton@amd.com>
Change-Id: Ibd41bada8220784f4b48f45efc776cf74ab0bfac
---
 lib/comgr/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/comgr/CMakeLists.txt b/lib/comgr/CMakeLists.txt
index 5362884..38efdb3 100644
--- a/lib/comgr/CMakeLists.txt
+++ b/lib/comgr/CMakeLists.txt
@@ -209,7 +209,7 @@ target_include_directories(amd_comgr
 set(AMD_COMGR_CONFIG_NAME amd_comgr-config.cmake)
 set(AMD_COMGR_TARGETS_NAME amd_comgr-targets.cmake)
 set(AMD_COMGR_VERSION_NAME amd_comgr-config-version.cmake)
-set(AMD_COMGR_PACKAGE_PREFIX lib/cmake/amd_comgr)
+set(AMD_COMGR_PACKAGE_PREFIX ${CMAKE_INSTALL_LIBDIR}/cmake/amd_comgr)
 
 # Generate the build-tree package.
 set(AMD_COMGR_PREFIX_CODE)
-- 
2.34.1

